micrometer-core/src/main/java/io/micrometer/core/ipc/http/HttpSender.java

```diff
@@ -304,6 +313,10 @@ public String body() {
             return body;
         }
 
+        public Map<String, List<String>> headers() {
```

Comment:
Since this isn't used anymore, we can get rid of these changes until we have a use for them, I think. If there is some use outside of supporting InfluxDB v2, we can consider them in a separate pull request.

Reply:
Fixed